tests/ci/unit_tests_staging.yml

```diff
@@ -0,0 +1,86 @@
+# unit_tests_staging.yml
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+#
+# use variable group name
+variables:
+- group: AzureKeyVaultSecrets
+
+#trigger:
+#- azure-pipelines-bz
+#  - master
+
+#pr:
+#- staging 
+  
+pool:
+ vmImage: 'ubuntu-16.04'
+
+steps:
+
+- task: UsePythonVersion@0
+  inputs:
+    versionSpec: '3.6'
+    architecture: 'x64'
+  displayName: 'Use Python 3.6'
+
+
+- script: | 
+   az login --service-principal -u $(ClientID) -p $(ClientSecret) --tenant $(TenantID)
+   
+  displayName: 'Login to Azure'
+
+- script: |
+   sed -i 's#"subscription_id": "<>"#"subscription_id": "$(SubscriptionID)"#g' ./tests/ci/config.json
+   echo my subscription is $(SubscriptionID)
+   cat ./tests/ci/config.json
+  displayName: 'replace subscription value'
+
+- script: 
+   sed -i 's#"tests/unit_or_smoke_int"#"tests/unit"#g' ./tests/ci/runpytest.py

```

Comment:
might be more straightforward to just pass these as arguments for submitpytest

Reply:
agree. again, I was asked to get something out now and then fix these items.

Reply:
deleted sed. Added needed parameters.